src/js/components/CreateServiceModalServicePickerOptionWrapper.js

```diff
@@ -0,0 +1,16 @@
+import {Hooks} from 'PluginSDK';
+import React from 'react';
+
+function ServicePickerOptionWrapper({children}) {
```

Comment:
Do we actually need this as a separate component? Couldn't this be part of `CreateServiceModalServicePickerOption`?